resources/metadata/whitesource.yaml

```diff
@@ -407,6 +407,14 @@ spec:
           - name: custom/whitesourceProjectNames
             type: "[]string"
   containers:
+    - image: devxci/mbtci:1.0.14
```

Comment:
Should we also add default images for npm and maven scan types? And perhaps we should add a comment to the Groovy-Wrapper where it assigns scanType from buildTool, that removing this would break the container-conditions in the Go-step.

Reply:
Yes. Makes sense.